src/qt/rpcconsole.cpp

```diff
@@ -462,13 +462,19 @@ void RPCConsole::clear()
     }
 
     // Set default style sheet
+    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont());
+    // Try to make fixed font adequately large on different OS
+    QString ptSize = QString("%1pt").arg(QFontInfo(QFont()).pointSize() * 8.5 / 9);
```

Comment:
@luke-jr Maybe you could do something like http://stackoverflow.com/a/12179548 to read the system default font. But I am not planning to do this within this PR, as this will likely change the text for the whole GUI.

I see only three options for this PR:
* Close
* Merge 28313b83fc1a16b5e6da2b868b5f297050ce8b05 (Monospace with "old" text-height)
* Merge 268b79ef0c54a3bff0b50e093b98b943e49a4939 (Same as above with constant scale factor)